### PR TITLE
CLIO-41709: Fix error thrown when null values are included in thTable

### PR DESCRIPTION
**Pull request by craigcarlyle**

### CODE REVIEW

File: themis_components/thTable/thTableSort.service.coffee [hunk 1]
```diff
@@ -9,10 +9,12 @@ TableSort = ->
       -compareResult
 
   compare = (a, b) ->
-    if typeof a in ["number", "boolean"]
+    if typeof a == "number"
```
Comment (JeffMarvin) [inline themis_components/thTable/thTableSort.service.coffee]: Technically should be `is` not `==` by our style guide. Same with below (15-17).

Comment (craigcarlyle) [inline themis_components/thTable/thTableSort.service.coffee]: Addressed.

File: themis_components/thTable/thTableSort.service.coffee [hunk 2]
```diff
@@ -9,10 +9,12 @@ TableSort = ->
       -compareResult
 
   compare = (a, b) ->
-    if typeof a in ["number", "boolean"]
+    if typeof a is "number"
       a - b
     else
-      a.localeCompare b
+      va = if a is null then '' else '' + a
```
Comment (seanhealy) [inline themis_components/thTable/thTableSort.service.coffee]: Also can we use variable names here that are a bit more descriptive. This is a bit hard to follow.

File: themis_components/thTable/thTableSort.service.coffee [hunk 3]
```diff
@@ -9,10 +9,12 @@ TableSort = ->
...
+      vb = if b is null then '' else '' + b
+      if va > vb then 1 else if va is vb then 0 else -1
```
Comment (seanhealy) [inline themis_components/thTable/thTableSort.service.coffee]: We should keep using `localeCompare` as it deals with a lot of edge cases for us.
